server/views/partials/header.njk

```diff
@@ -1,4 +1,4 @@
-<header class="hmpps-header" role="banner">
+<header class="app-header" role="banner">
```

Comment:
As discussed, there are conflicting styles with the DPS fallback header and our original header. `app-` is the usual convention used when its a local application component/style.